package.json

```diff
@@ -30,7 +30,14 @@
     "@babel/register": "^7.0.0",
     "babel-loader": "^8.0.0",
     "coveralls": "^2.13.0",
-    "ocular-dev-tools": "0.0.25",
+    "eslint": "^3.19.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-config-uber-es2015": "^3.1.2",
+    "eslint-config-uber-jsx": "^3.3.3",
+    "eslint-import-resolver-webpack": "^0.11.1",
+    "eslint-plugin-import": "^2.16.0",
```

Comment:
This is a lot of dependencies, please confirm that they are really used. E.g. where are we using plugin-import? Isn't that for workers?